osquery/tables/yara/yara.cpp

```diff
@@ -92,7 +114,7 @@ QueryData genYara(QueryContext& context) {
         if (status.ok()) {
           for (const auto& resolved : patterns) {
             // Check that each resolved path is readable.
-            if (isReadable(resolved)) {
+            if (isReadable(resolved) && !shouldSkipFile(resolved)) {
```

Comment:
Is there an existing API (boost or osquery) to help identify “special” files?

Reply:
not that I see.  There's platformAccessible, but not what is needed here.